### PR TITLE
Update readmes and fix out of date things found along the way

### LINKED ISSUE #2333: Review and update data README
**Is your feature request related to a problem? Please describe.**
There is a README about our test data at https://github.com/bbc/simorgh/blob/latest/data/README.md however it is currently not very clear. It also needs more information about the assets we have created for audio-video content.

We should review this document and update it so that it's clear, accurate and comprehensive. This will make life easier for testers and developers to avoid having to ask around for test IDs/URLs.

**Describe the solution you'd like**
As above. In particular:

- TBD

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by jamesbhobbs**
Resolves #2333 #2229 #562 and my general statement "our documentation is completely unmaintainable in its present state"

Evidence for: "our documentation is completely unmaintainable in its present state" is how easily and how many errors I found and addressed here.

**Overall change:** Revises all documentations with minor fixes implemented found along the way

**Code changes:**
- Change all readme and documentation files to be in an indexed format that is shared between the files (there are only 5 remaining)
- Major updates to the primary readme (lots was out of date), aside from just corrections, it also downplays articles and incorporates the simorgh overview that was in a separate file.
- Removes all of the cypress tests for storybook (because chromaticQA is doing the same testing now)
- Corrects and e2e test description in cypress/integration/metaArticleSpec.js
- Removes the data README (as it was out of date and is impossible to keep up to date as we add hundreds of datasets) - then fixes tests broken by doing this
- Rmvs creating components README that should be done in psammead, explaining it in a new way then references psammead documentation would be reasonable but removing the file increases clarity for now.
- Remove developer help, local setup files - this type of thing needs to be within the local dev instructions, an orphaned file no one will see isn't helpful
- Remove manual testing file - everything in it can now be done in chromatic QA
- Corrected docs/Project-Board-Guide.md as it referenced a column that no longer exists
- Removed spacing unit and webpack decision records. Explaining the choice of webpack and REMs is 100% redundant. We don't keep Architectural Decision Records (ADRs) anyhow.
- Correct all instances of `http://localhost:7080` to `http://localhost.bbc.com:7080`
- Removed the readmes from the root of components and containers. I actually wrote a lot of these, but they're now extremely buried. And honestly all they say is what a container is. Not worth having.

---

- [x] I have assigned myself to this PR and the corresponding issues

No testing needed, the changes are to tests and docs only

### CODE REVIEW

**Comment by jamesbhobbs** [comment]
I cannot see why this is failing in jenkins but passing in travis, I suspect I've missed removing a reference to the e2e test for storybook, or it could be that the replacement of localhost with localhost.bbc.com is crashing only in jenkins, but I see no console output to explain why. Help appreciated.

File: dataValidator/helpers/dataLoader/readScenario.test.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ describe('readScenario helper', () => {
   });
 
   it('should ignore files that are no json format', () => {
```
Comment (chris-hinds) [inline dataValidator/helpers/dataLoader/readScenario.test.js]:
```suggestion
  it('should ignore files that are not in json format', () => {
```

File: docs/Code-Standards.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+## Documentation index
+Please familiarise yourself with our:
+- [Code of conduct](https://github.com/bbc/simorgh/blob/latest/.github/CODE_OF_CONDUCT.md)
+- [Code Standards](https://github.com/bbc/simorgh/blob/latest/docs/Code-Standards.md) (you are here)
+- [Contributing guidelines](https://github.com/bbc/simorgh/blob/latest/CONTRIBUTING.md)
+- [Github Project Board Guide](https://github.com/bbc/simorgh/blob/latest/docs/Project-Board-Guide.md)
+- [Primary README](https://github.com/bbc/simorgh/blob/latest/README.md)
+
+## Overarching code standards
 - We are following an AMP-first approach therefore all page components must be AMP compatible. [Example of component with AMP Compatibility](https://github.com/bbc/psammead/tree/latest/packages/components/psammead-image) – the component is included in an AMP page and passes AMP validation.
```
Comment (chris-hinds) [inline docs/Code-Standards.md:13]: Would it beneficial to link to the AMP site here for those who may not know much about the AMP spec?

https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml/

File: src/server/assets/assetsFilter/index.test.js [hunk 3]
```diff
@@ -8,21 +8,21 @@ describe('assetsFilter', () => {
...
     it('should order assets service bundle, vendor, main on local', () => {
       const input = [
-        'http://localhost:7080/static/js/yoruba-31ecd969.d4952cef.js',
-        'http://localhost:7080/static/js/main-d0ae3f07.e24ffe78.js',
-        'http://localhost:7080/static/js/vendor-f9ca8911.836b5376.js',
-        'http://localhost:7080/static/js/pidgin-31ecd969.652d66cc.js',
-        'http://localhost:7080/static/js/news-31ecd969.c141cfdc.js',
-        'http://localhost:7080/static/js/vendor-b23bnb22.nmn32mn2.js',
-        'http://localhost:7080/static/js/vendor-1f20a385.ff6d3f55.js',
+        'http://localhost.bbc.com:7080/static/js/yoruba-31ecd969.d4952cef.js',
```
Comment (chris-hinds) [inline src/server/assets/assetsFilter/index.test.js]: This is a lot of times to declare the localhost URL. Is it worth declaring this as a const at the top of this page or even imported from the env files?

Out of scope for this PR but we could raise a new issue.

File: README.md [hunk 4]
```diff
@@ -2,25 +2,95 @@
 
 [![Build Status](https://travis-ci.org/bbc/simorgh.svg?branch=latest)](https://travis-ci.org/bbc/simorgh) [![Test Coverage](https://api.codeclimate.com/v1/badges/cbca275e184057982f27/test_coverage)](https://codeclimate.com/github/bbc/simorgh/test_coverage) [![Known Vulnerabilities](https://snyk.io/test/github/bbc/simorgh/badge.svg)](https://snyk.io/test/github/bbc/simorgh) [![Maintainability](https://api.codeclimate.com/v1/badges/cbca275e184057982f27/maintainability)](https://codeclimate.com/github/bbc/simorgh/maintainability) [![Storybook](https://cdn.jsdelivr.net/gh/storybooks/brand@master/badge/badge-storybook.svg)](https://bbc.github.io/simorgh/)
 
-Named Simorgh after the Persian mythological bird. The Simorgh is the amalgam of many birds (and in some accounts other animals) into one.
+The BBC websites are being rebuilt from the bottom up as ReactJS based Single Page Applications, that will (eventually) also be fully fledged Progressive Web Apps. This application also builds Accelerated Mobile Pages for every regular HTML page that it renders.
```
Comment (chris-hinds) [inline README.md:5]: Should this be specific to worldservice? and not the rest of the BBC?

File: README.md [hunk 5]
```diff
@@ -2,25 +2,95 @@
...
+- [Contributing guidelines](https://github.com/bbc/simorgh/blob/latest/CONTRIBUTING.md)
+- [Github Project Board Guide](https://github.com/bbc/simorgh/blob/latest/docs/Project-Board-Guide.md)
+- [Primary README](https://github.com/bbc/simorgh/blob/latest/README.md) (you are here)
+
+## Simorgh Overview
+### A High Level User Journey
+
+#### The initial page load - Server Side Render (SSR)
+
+A request to a BBC article (https://www.bbc.co.uk/news/articles/clldg965yzjo) is passed on to the Simorgh application from Mozart.
```
Comment (chris-hinds) [inline README.md]: we should either say what `Mozart` is or just remove the mention of Mozart from the docs.

File: README.md [hunk 6]
```diff
@@ -2,25 +2,95 @@
...
+
+The request matches a route in our express server using a regex match (`articleRegexPath` || `frontPageRegexPath`). If the URL matches the pre-defined regex pattern for an article or a front page we fetch some params from the route using the `getRouteProps` function. This returns the service, isAmp, route and match. Route is a react-router route that defines a method to fetch the initial JSON used to render the page and the react container in which to render i.e. `ArticleContainer`, this is typically called `getInitialData`
```
Comment (chris-hinds) [inline README.md]:
```suggestion
The request matches a route in our express server using a regex match (`articleRegexPath` || `frontPageRegexPath`). If the URL matches the pre-defined regex pattern for an article or a front page we fetch some params from the route using the `getRouteProps` function. This returns the service, isAmp, route and match properties. Route is a react-router route that defines a method to fetch the initial JSON used to render the page and the react container in which to render i.e. `ArticleContainer`, this is typically called `getInitialData`
```

File: README.md [hunk 7]
```diff
@@ -2,25 +2,95 @@
...
+
+Now that the raw HTML has been downloaded, the client-side JS file kicks in and hydrates the initial response with the client side application. During this process react uses the initial JSON payload (available on the global window object `SIMORGH_DATA`) to hydrate the original markup returned by ReactDOMServer. React expects that the rendered content is identical between the server and the client (This is why we send the initial JSON payload with the SSR page, so the hydration phase runs with the same data that the server render used).
```
Comment (chris-hinds) [inline README.md:32]: Maybe append to this last paragraph that the use of the hydrate method and the SIMORGH_DATA object should result in no visual change to the user?

File: README.md [hunk 8]
```diff
@@ -2,25 +2,95 @@
...
+
+#### Onward Journeys - Client Side Render (CSR)
+
+Now that the SPA is bootstrapped on the users device, onward journeys to further Simorgh pages are captured by react-router and rendered on entirely on the client.
```
Comment (chris-hinds) [inline README.md]:
```suggestion
Now that the SPA is bootstrapped on the users device, onward journeys to further Simorgh pages are captured by react-router and rendered entirely on the client.
```

File: README.md [hunk 9]
```diff
@@ -2,25 +2,95 @@
...
+
+Clicking on an onward journey triggers the `useEffect` hook in the main App container as the URL path has changed. This effect uses `getRouteProps` again to match the new path against a react-router route, gaining access to the; service and isAmp props, the react container to render and the `getInitialData` function.
+
+The local state is now updated setting `loading` to `true` and `data` to `null`. This update to state triggers the App container to re-render passing the state values as props to the main container, in this case either ArticlesContainer or FrontPageContainer. These containers compose a set of higher order components (HOC) and each one handles the values from a given props. Currently the loading prop is set to `true` so the `withLoading` HOC will return the loading component showing a visual loading state to the user.
```
Comment (chris-hinds) [inline README.md]:
```suggestion
The local state is now updated, setting `loading` to `true` and `data` to `null`. This update to state triggers the App container to re-render passing the state values as props to the main container, in this case either ArticlesContainer or FrontPageContainer. These containers compose a set of higher order components (HOC) and each one handles the values from a given props. Currently the loading prop is set to `true` so the `withLoading` HOC will return the loading component showing a visual loading state to the user.
```

Comment (chris-hinds) [reply README.md]: These commit suggestions aren't easy to see the changes. This one just added a comma.

File: README.md [hunk 10]
```diff
@@ -2,25 +2,95 @@
...
+
+During this time the App container has finished fetching the JSON payload for the onward journey and the state is updated again. Loading is now set to false and data is set to the returned JSON data.
+
+The routes container is rendered again, this time loading is set to false so the `withData` HOC can run some validation against the JSON payload then return either an error components of the original page container e.g. ArticleContainer passing in the JSON data with the pageData prop.
```
Comment (chris-hinds) [inline README.md]:
```suggestion
The routes container is rendered again, this time loading is set to false so the `withData` HOC can run some validation against the JSON payload and return either an error component or the original page container e.g. ArticleContainer passing in the JSON data with the pageData prop.
```

File: README.md [hunk 11]
```diff
@@ -2,25 +2,95 @@
...
+
+The ArticleMain container will iterate over each JSON block, match it against its corresponding react container and pass the data via props. These containers are where the logic for rendering each block type sits. It is at this point where we use the installed frontend components from the Psammead component library. For example the Image container will import the Figure container, and Figure will import and use the psammead-image and the psammead-image-placeholder components. An image on an article will generally have a caption, so the Figure container will import the caption container which may include more frontend components from Psammead to render a caption on top of the image.
+
+This process is repeated for each block within an article, ultimately rendering the main body of a news article using a combination of React containers for the business logic and React components for the frontend markup.
+
+### A Page Render Lifecycle
+
+Each render is passed through a set of HOC's to enhance the page, these HOC's are;
```
Comment (chris-hinds) [inline README.md]:
```suggestion
Each render is passed through a set of HOC's (Higher Order Components) to enhance the page, these HOC's are;
```

File: README.md [hunk 12]
```diff
@@ -56,6 +130,8 @@ We are also serving AMP HTML pages at the route `/news/articles/:id.amp` [https:
 - [http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po.amp](http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po.amp)
 - [http://localhost.bbc.com:7080/persian/articles/c4vlle3q337o.amp](http://localhost.bbc.com:7080/persian/articles/c4vlle3q337o.amp).
 
+### Front pages
+
 World Service front pages are served in the format `/:service` where `service` represents a World Service site:
```
Comment (chris-hinds) [inline README.md:135]: We could link to the languages page here https://www.bbc.com/ws/languages to show how languages we will be working with?

File: cypress/integration/metaArticleSpec.js [hunk 13]
```diff
@@ -23,7 +23,7 @@ Object.keys(services)
         });
       });
 
-      it('should have a nofollow meta tag', () => {
+      it('should have a correct robot meta tags', () => {
```
Comment (chris-hinds) [inline cypress/integration/metaArticleSpec.js]:
```suggestion
      it('should have a correct robot meta tag', () => {
```

**Review by chris-hinds — CHANGES_REQUESTED** [review]

**Comment by chris-hinds** [comment]
> I cannot see why this is failing in jenkins but passing in travis, I suspect I've missed removing a reference to the e2e test for storybook, or it could be that the replacement of localhost with localhost.bbc.com is crashing only in jenkins, but I see no console output to explain why. Help appreciated.

@benjaminhobbs 
Looks like Jenkins CI is failing as you need to remove the command to run the cypress storybook tests from the [Makefile](https://github.com/bbc/simorgh/blob/latest/Makefile#L12)

File: .github/CODE_OF_CONDUCT.md [hunk 14]
```diff
@@ -1,5 +1,13 @@
 # Contributor Covenant Code of Conduct
 
+## Documentation index
```
Comment (sareh) [inline .github/CODE_OF_CONDUCT.md]: This index doesn't need to be in the code of conduct file. Its presence here currently implies that if you don't abide by code standards you could be in violation of the CoC, which really isn't the case!

File: README.md [hunk 15]
```diff
@@ -2,25 +2,95 @@
...
+
+## Documentation index
+Please familiarise yourself with our:
+- [Code of conduct](https://github.com/bbc/simorgh/blob/latest/.github/CODE_OF_CONDUCT.md)
+- [Code Standards](https://github.com/bbc/simorgh/blob/latest/docs/Code-Standards.md)
+- [Contributing guidelines](https://github.com/bbc/simorgh/blob/latest/CONTRIBUTING.md)
+- [Github Project Board Guide](https://github.com/bbc/simorgh/blob/latest/docs/Project-Board-Guide.md)
+- [Primary README](https://github.com/bbc/simorgh/blob/latest/README.md) (you are here)
+
+## Simorgh Overview
```
Comment (sareh) [inline README.md:19]: I appreciate having the Simorgh overview in one place rather than spread across several markdown files. However, I don't believe the main readme is the best place for this. If you look at most successful open source projects, their initial readme is very succinct, and just points you to the relevant getting-started page or background-to-the-application page.

I believe Simorgh could greatly benefit from this same change.

File: data/README.md [hunk 16]
```diff
@@ -1,52 +0,0 @@
...
-| 31       | Home: News | Headline, Text (1 para), Subheadline, Video (portrait) | No tags | Test |
-| 32       | Home: News | Headline, Subheadline, Video, Subheadline, Audio | No tags | Test |
-| 33       | Home: News | Headline, Text (1 para), Subheadline, Video, Subheadline, Video, Subheadline, Video, Subheadline, Video | No tags | Test |
-| 34       | Home: News | Headline, Subheadline, Video, Subheadline, Video, Subheadline, Video, Subheadline, Video, Subheadline, Audio, Subheadline, Audio, Subheadline, Audio | No tags | Test |
-
-## Audio/Video test assets
-
-| ID | Description |
-| -----------: | ------------- |
-| c3wmq4d1y3wo | TEST asset containing multiple AV scenarios |
```
Comment (sareh) [inline data/README.md:51]: I understand deleting this document since it is no longer maintainable now that we have dozens more World Service assets. However, it should be replaced with a short explanation in the Contributing.md about how to add/find relevant local fixture files. Eg. How to find an asset with a video block.

File: docs/Creating-Components.md [hunk 17]
```diff
@@ -1,150 +0,0 @@
...
-        type:
-          enum:
-            - blockquote
-          type: string
-```
-
-The schema check currently only happens on local data. Eventually, we hope to run the schema validator at runtime.
-
-### Update some local fixture data
-Pick a JSON file under `data/news/articles/[id].json`, and:
```
Comment (sareh) [inline docs/Creating-Components.md:102]: I understand deleting the components information, since we are now creating components in Psammead. It would be useful to keep the 'how to update fixture data & schema' sections within our Simorgh documentation, in a more useful location, likely the Contributing.md file.

File: docs/Local-Setup.md [hunk 18]
```diff
@@ -1,13 +0,0 @@
-# Local setup
-
-## Using Visual Studio Code editor
-You can install the ESLint extension
```
Comment (sareh) [inline docs/Local-Setup.md:4]: Keeping this in an FAQ section of the Contributing.md or a GettingStarted.md would be useful, especially for new users of Simorgh and it also helps our new starters.

File: docs/Project-Board-Guide.md [hunk 19]
```diff
@@ -4,6 +4,14 @@ Overview of a Github Project: https://help.github.com/en/articles/about-project-
 
 BBC's Simorgh Project board: https://github.com/orgs/bbc/projects/20
 
+## Documentation index
+Please familiarise yourself with our:
```
Comment (sareh) [inline docs/Project-Board-Guide.md:8]: I like the documentation index you've created. Very useful for orienting yourself within the docs and knowing what other resources are available.

File: docs/Spacing-Units.md [hunk 20]
```diff
@@ -1,81 +0,0 @@
-# REMs vs EMs for spacing
-REMs and EMs can be used for spacing (padding/margins/line height). We chose REMs on the basis of clarity and no perceptible difference in rendering.
```
Comment (sareh) [inline docs/Spacing-Units.md:2]: This Readme is now also in Psammead, so it is good that is is being removed from here.

File: docs/Webpack.md [hunk 21]
```diff
@@ -1,36 +0,0 @@
-# Webpack
-
```
Comment (sareh) [inline docs/Webpack.md:2]: I agree that this webpack document isn't adding much value and should be removed. We should separately discuss how we will write up technical decisions (in an ADR format or otherwise) within the documentation of this application.

File: src/app/components/README.md [hunk 22]
```diff
@@ -1,11 +0,0 @@
-# Components
-
-## Overview
```
Comment (sareh) [inline src/app/components/README.md:3]:
```suggestion
We are now creating new components within [Psammead] (https://github.com/bbc/psammead), our component library. We are in the process of migrating all existing components out of Simorgh and into Psammead. If you need to add a component to be used in this application, please do so within Psammead. 
```

File: src/app/containers/README.md [hunk 23]
```diff
@@ -1,36 +0,0 @@
-# Containers
-
-## Overview
```
Comment (sareh) [inline src/app/containers/README.md:3]:
```suggestion
We are now creating containers that are agnostic to data source within [Psammead] (https://github.com/bbc/psammead), our component library. We are in the process of migrating all existing reusable containers out of Simorgh and into Psammead. If you need to add a container to be used in this application, please first open an issue to discuss where it should best be placed, within Psammead or Simorgh. 
```

File: src/app/containers/README.md [hunk 24]
```diff
@@ -1,36 +0,0 @@
...
-
-```js
-/*
-* The webpack.ContextReplacementPlugin allows us to load only
-* the specific locales we require into our code. By using this
-* plugin we can reference `moment/locale/en.js` in the files
-* we need it, without importing it in every file or importing
-* every locale.
-*/
-new webpack.ContextReplacementPlugin(/moment[/\\]locale$/, /en-gb/),
```
Comment (sareh) [inline src/app/containers/README.md:27]: We should retain the information regarding webpack plugin for Moment next to the timestamp containers. But it should be rewritten to be useful for consumers of containers and contributors to Simorgh. Happy for this to be in a separate PR.

File: docs/Developer-Help.md [hunk 25]
```diff
@@ -1,23 +0,0 @@
...
-Webpack creates two bundles, the first being `server` that has an entrypoint of `/src/index.js` and the second being `client` which has an entry point of `/src/client.js`
-
-The entry points to the application in terms of code execution are as follows: 
-- `npm run build:ci` executes the webpack configs
-- `webpack.config.js` is executed and then executes both `webpack.config.client.js` and `webpack.config.server.js`
-- `webpack.config.client.js` then executes `src/client.js` 
-- `webpack.config.server.js` then executes `src/index.js` 
-- `src/index.js` then executes `src/server/index.jsx`
-
-## `.env` is showing in my `git status`
```
Comment (sareh) [inline docs/Developer-Help.md:16]: This issue is often seen in new contributors' initial PRs. So to improve the onboarding experience, I'd suggest keeping this paragraph within an FAQ section of a Getting-Started.md or Contributing.md.

**Review by sareh — COMMENTED** [review]
Great work tidying up the documentation. I've left a few comments below.

File: README.md (hunk 4 above)
Comment (jamesbhobbs) [reply README.md:5]: Deliberately was talking about all of DPG without naming DPG because no one will know what it is

File: README.md (hunk 15 above)
Comment (jamesbhobbs) [reply README.md:19]: So moving all the local dev and simorgh overview to another doc? Or something else?

File: data/README.md (hunk 16 above)
Comment (jamesbhobbs) [reply data/README.md:51]: I find it debatable whether this exists now as this document is very unclear, but I agree in principle. Would a few lines explaining the data folder and with example of how to search within it suffice?

File: docs/Local-Setup.md (hunk 18 above)
Comment (jamesbhobbs) [reply docs/Local-Setup.md:4]: I think it is better placed in our internal Dropbox docs, it seems wrong for an OSS project to prefer one editor over another and give support for it.

File: src/app/components/README.md (hunk 22 above)
Comment (jamesbhobbs) [reply src/app/components/README.md:3]: Will remove them here then have another PR to add a very simple readme to the base of every major dir

File: src/app/containers/README.md (hunk 23 above)
Comment (jamesbhobbs) [reply src/app/containers/README.md:3]: Will remove them here then have another PR to add a very simple readme to the base of every major dir

File: docs/Code-Standards.md (hunk 2 above)
Comment (jamesbhobbs) [reply docs/Code-Standards.md:13]: Done

File: README.md (hunk 5 above)
Comment (jamesbhobbs) [reply README.md]: Done

File: src/server/assets/assetsFilter/index.test.js (hunk 3 above)
Comment (jamesbhobbs) [reply src/server/assets/assetsFilter/index.test.js]: Interesting I actually prefer duplication in test files. If you'd like this refactored please open an issue. I like it how it is, but don't object to it being refactored.

Comment (chris-hinds) [reply src/server/assets/assetsFilter/index.test.js]: Will raise a new issue and let others discuss 👍

File: README.md (hunk 12 above)
Comment (jamesbhobbs) [reply README.md:135]: Above in same file I've linked to https://github.com/bbc/simorgh/wiki/Simorgh-Pages to show progress, I think until we've done a lot of the services it's clearer to link to this link than that one as it would most likely lead people to a page produced by a different application.

File: .github/CODE_OF_CONDUCT.md (hunk 14 above)
Comment (jamesbhobbs) [reply .github/CODE_OF_CONDUCT.md]: Fixed

File: src/app/components/README.md (hunk 22 above)
Comment (jamesbhobbs) [reply src/app/components/README.md:3]: https://github.com/bbc/simorgh/issues/2873

File: src/app/containers/README.md (hunk 23 above)
Comment (jamesbhobbs) [reply src/app/containers/README.md:3]: https://github.com/bbc/simorgh/issues/2873

File: README.md (hunk 15 above)
Comment (sareh) [reply README.md:19]: Yes, `Simorgh-Overview.md` for the overview and local dev setup to a `Getting-Started.md`. Or you could have a single markdown file with these as two headings.

File: data/README.md (hunk 16 above)
Comment (sareh) [reply data/README.md:51]: Yes, that'd be good.

File: docs/Local-Setup.md (hunk 18 above)
Comment (sareh) [reply docs/Local-Setup.md:4]: Agreed.

File: docs/Developer-Help.md (hunk 25 above)
Comment (jamesbhobbs) [reply docs/Developer-Help.md:16]: Copied into contributing.md

File: src/app/containers/README.md (hunk 24 above)
Comment (jamesbhobbs) [reply src/app/containers/README.md:27]: I don't know how to write the issue to do this as I don't see how it's helpful. Aren't we just saying that once it's in psammead we should make clear that we recommend using this plugin? Doing it before then doesn't make any sense to me, simorgh has no consumers.

File: data/README.md (hunk 16 above)
Comment (jamesbhobbs) [reply data/README.md:51]: Added a draft under title Fixture data in contributing.

File: docs/Creating-Components.md (hunk 17 above)
Comment (jamesbhobbs) [reply docs/Creating-Components.md:102]: There's scope ambiguity in this suggestion, which sections should be kept? I suggset the location is no best under Fixture data in contributing.

File: README.md (hunk 7 above)
Comment (jamesbhobbs) [reply README.md:32]: I'm happy yo add this, can you add this in the form of a suggested change for me to accept please. It'll then be quicker to agree the copy

File: README.md (hunk 15 above)
Comment (jamesbhobbs) [reply README.md:19]: I mostly agree for style of readability (my hesitation is on maintainability), but you're actually already suggesting some duplication - which I think shows the fewer files the more maintainable the docs are. Specifically we have getting started at the top of contributing.md already.

I suggest we merge this PR as a major change (which has many corrections) then can revisit pulling things out of the primary readme, adding them to the documentation index and being clear on the point of each file. At the moment this primary readme is a catchall.

IMO the primary readme would ideally have all the overview info and link clearly to a single contributing/getting-started file. That way it's clear to a reader and writer which things go where. Elsewise it's always a judgment call.

File: docs/Creating-Components.md (hunk 17 above)
Comment (sareh) [reply docs/Creating-Components.md:102]: Contributing.md - under a new heading `## Local fixture data`, which can have the sections:
`### Updating local fixture data`
`### Updating the schema`

File: src/app/containers/README.md (hunk 24 above)
Comment (sareh) [reply src/app/containers/README.md:27]: Sure, we can add it to the relevant container once we move it into Psammead. It lives in git history if someone needs to refer to it at that point.

File: docs/Creating-Components.md (hunk 17 above)
Comment (sareh) [reply docs/Creating-Components.md:102]: Actually, I see you've already got a `## Fixture data` section, so it can live there https://github.com/bbc/simorgh/pull/2870/files#diff-6a3371457528722a734f3c51d9238c13R105

Comment (jamesbhobbs) [reply docs/Creating-Components.md:102]: Done

**Review by sareh — APPROVED** [review]
👍 Thanks for making all the changes! It's much more organised than it was before.

File: README.md (hunk 15 above)
Comment (sareh) [reply README.md:19]: Agreed that the changes made here are sufficient for this set of changes and that we can reassess in future to make the files more readable for new contributors to the repo.

**Review by chris-hinds — APPROVED** [review]
